src/app/loaders/express/index.ts

```diff
@@ -108,40 +107,16 @@ const loadExpressApp = async (connection: Connection) => {
 
   app.use(helmetMiddlewares())
 
-  // !!!!! DO NOT CHANGE THE ORDER OF THE NEXT 3 LINES !!!!!
-  // The first line redirects requests to /public/fonts to
-  // ./dist/frontend/fonts. After that, nocache() ensures that
-  // cache headers are not set on requests for fonts, which ensures that
-  // fonts are shown correctly on IE11.
-  // The last line redirects requests to /public to ./dist/frontend,
-  // with cache headers set normally.
-  app.use(
-    '/public/fonts',
-    express.static(path.resolve('./dist/frontend/fonts')),
-  )
-
-  app.use(nocache()) // Add headers to prevent browser caching front-end code
-
   // Generate UUID for request and add it to X-Request-Id header
   app.use(addRequestId())
 
-  // Setting the app static folder
-  app.use('/public', express.static(path.resolve('./dist/frontend')))
-
-  // Point crawlers to our robots.txt
-  app.use(
```

Comment:
do we not need to keep the functionality in this line?

Reply:
We do! Opened an issue #2729 to track